portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsRichTextField.testcase

```diff
@@ -54,14 +54,58 @@ definition {
 	}
 
 	@description = "Verify that a Rich Text Field can be duplicated"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeDuplicated {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133113 CanBeDuplicated pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Rich Text");
+
+		FormFields.editHelpText(
+			fieldHelpText = "Help Text Test",
+			fieldName = "tip");
+
+		Form.gotoAdvancedTab();
+
+		CKEditor.addContentInWebContent(
+			content = "Rich Text Test",
```

Comment:
Hey @PauloVAS can you change this CKEditor macro to a Forms macro please?

Reply:
Ok, starting now. Thanks